### DISCUSSION TITLE
How to declare and initialize a Battery group correctlly?

### DISCUSSION THREAD

**Discussion by lipper** [rlogiacco/BatterySense#20] [Q&A] [answered]
Battery[] battery = new Battery[12];

build error: expected unqualified-id before '{' token 

Very kind of you , thanks .

**Comment by lipper** [comment]
Sorry, I found the reason : 
   Because the construction function has parameters, so I could not declare a class group like that:
Battery[] battery = new Battery[12];
If must, I should add a empty construction function !
Whatever , thank you for create a BatterySense ！

**Comment by rlogiacco** [comment, marked as answer]
`Battery[12] batteries;`

Arduino programming is not Java programming....
